### ISSUE TITLE
MO 2.0.7 Mod renaming bug Left pane

### ISSUE THREAD

**Issue by GSDFan** [TanninOne/modorganizer#889]
If you rename a mod in MO2.0.7 it will become deactivated and will be moved to the highest priority position in the list. This will not happen in V1.3.11 where it will stay activated and at the same priority.

**Comment by michaelrw10** [comment]
yeah this has been happening for a long time in mo2.. thought it was already reported.. maybe not
